packages/fast-tooling-react/src/form/custom-controls/control.align.tsx

```diff
@@ -1,95 +1,85 @@
 import React from "react";
 import manageJss, { ManagedJSSProps } from "@microsoft/fast-jss-manager-react";
 import { ManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
-import { CustomFormControlProps } from "../controls/control.props";
-import styles from "./control.align.style";
-import { AlignFormControlClassNameContract } from "./control.align.props";
-import BaseFormControl from "../controls/template.control.abstract";
+import styles, { AlignControlClassNameContract } from "./control.align.style";
+import { AlignControlProps, AlignDirection } from "./control.align.props";
+import { classNames } from "@microsoft/fast-web-utilities";
 
 /**
- * Schema form component definition
- * @extends React.Component
+ * Custom form control definition
  */
 /* tslint:disable-next-line */
-class AlignFormControl extends BaseFormControl<
-    CustomFormControlProps & ManagedClasses<AlignFormControlClassNameContract>,
+class AlignControl extends React.Component<
```

Comment:
should all of this be scoped to *vertical* alignment?

Reply:
Actually no, the idea was for object alignment so this should be expanded to include things like left and right, we just haven't gotten around to including them yet